clients/ui/frontend/src/style/MUI-theme.scss

```diff
@@ -447,15 +460,19 @@
 }
 
 .pf-v6-c-menu-toggle.pf-m-plain {
+  // Needs clarity --pf-v6-c-menu-toggle--m-plain--BorderRadius does not exist
```

Comment:
Ah, this is the right line, but it doesn't seem a new addition, regardless of that, a 50% would make that weird shape in the button